.bazelrc

```diff
@@ -12,6 +12,13 @@ build --host_force_python=PY2
 build --incompatible_use_python_toolchains
 common --incompatible_string_join_requires_strings
 
+# TODO(gregmagolan): remove the following flags once @io_bazel_rules_go is compatible with Bazel 0.27.0
+build --noincompatible_require_ctx_in_configure_features
+build --noincompatible_no_support_tools_in_action_inputs
+
+# Faster TypeScript compiles
```

Comment:
I thought we would remove all of these now that recent bazel has the strategy_list_selection flag that selects worker mode automatically in actions that support it?

Reply:
👍 Got pulled in from one of the old nested workspaces but you're right that it can be removed